pull-requests.md

```diff
@@ -22,6 +16,22 @@ The content changes go through the following stages before they are published on
 2. **Review:** Your changes are reviewed by [MDN members and volunteers](#pull-request-review-process).
 3. **Publishing:** Updated content goes live within a day of merging via a site rebuild once every 24 hours.
 
+## Before you start
+
+Make sure that you're not making changes that someone else is already working on.
+You can find tasks under 'Issues' in any of the [MDN repositories](https://github.com/orgs/mdn/repositories) (for example, [`mdn/content` issues](https://github.com/mdn/content/issues)) and our [public GitHub project boards](https://github.com/orgs/mdn/projects).
```

Comment:
Forgot to mention `good first issues` tag.

Reply:
Not sure if that label is only in `mdn/content`. So we might want to say that as well?

Reply:
I think we can omit being specific for the label in just `mdn/content` for now as we'll likely want to add that everywhere else as well. It's a pattern to [use this label in repos](https://www.kubernetes.dev/docs/guide/help-wanted/) to direct new contributors that I think we can use in other places